elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -378,11 +380,14 @@ export class PeerPool extends EventEmitter {
 	public triggerNewConnections(
 		peers: ReadonlyArray<P2PDiscoveredPeerInfo>,
 	): void {
-		const peersCount = this.getPeersCountByKind();
+		const disconnectPeers = peers.filter(
+			peer => !this._peerMap.has(constructPeerIdFromPeerInfo(peer)),
+		);
+		const { outbound } = this.getPeersCountByKind();
```

Comment:
not related to this PR, but `getPeersCountByKind ` seems to be bad function name.
It seems like it takes the `kind` as input, but it does not